src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionActivateHandler.java

```diff
@@ -209,8 +215,16 @@ public Mono<Tuple2<Mono<BaseTransactionEvent<?>>, String>> handle(
                         .collectList()
                         .flatMap(
                                 paymentRequestInfos -> jwtTokenUtils
-                                        .generateToken(transactionId, command.getData().orderId())
+                                        .generateToken(
+                                                ecommerceSigningKey,
+                                                jwtEcommerceValidityTimeInSeconds,
+                                                new Claims(transactionId, command.getData().orderId(), null)
+                                        )
                                         .map(generatedToken -> Tuples.of(generatedToken, paymentRequestInfos))
```

Comment:
Remove map and use only fold
```suggestion
                                        
                                        .fold(
                                                Mono::error,
                                                generatedToken -> Mono.just( Tuples.of(generatedToken, paymentRequestInfos)
                                        )
```